notebooks/qg/hybridqg_inference.ipynb

```diff
@@ -0,0 +1,377 @@
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "d3a5c821e071448fbb03c5ffb380710f",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "Downloading:   0%|          | 0.00/892M [00:00<?, ?B/s]"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "torch.Size([32105, 768]) torch.Size([32105, 768])\n"
+     ]
+    },
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "ed1748f44f66444c8c441a4ee1c2e2cb",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "Downloading:   0%|          | 0.00/1.93k [00:00<?, ?B/s]"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "47aebbc7f8da43a9a2ce383ddb214573",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "Downloading:   0%|          | 0.00/792k [00:00<?, ?B/s]"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "b3d7583151814cad872a51d51768f2ad",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "Downloading:   0%|          | 0.00/2.42M [00:00<?, ?B/s]"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "7ca597a80ee44bf495a5a0fad6e55b0a",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "Downloading:   0%|          | 0.00/98.0 [00:00<?, ?B/s]"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "f391fe4348ff4a99b39f1cbbd3670524",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "Downloading:   0%|          | 0.00/1.79k [00:00<?, ?B/s]"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "torch.Size([32105, 768]) torch.Size([32105, 768])\n"
+     ]
+    }
+   ],
+   "source": [
+    "#model_name = 'PrimeQA/t5-base-table-question-generator'\n",
+    "model_name = 'PrimeQA/t5-base-hybrid-question-generator'\n",
+    "hybrid_qg_model = QGModel(model_name, modality='hybrid', lang='en')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "984e88f0",
+   "metadata": {},
+   "source": [
+    "<br>\n",
+    "\n",
+    "## Sample table\n",
```

Comment:
Can you show this table in table form as well? (e.g. an image of the table)

Reply:
Do you mean just the table part or both table and text into one single table? The latter might not be possible due to large text passages.

Reply:
I meant just the table part, if this is doable.